```
Error running mint automation
+ mkdir -p 8641-7328d34/gopath/src/github.com/minio
+ git -C 8641-7328d34/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 8641-7328d34/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 8641-7328d34/gopath/src/github.com/minio/minio fetch --quiet minio pull/8641/head:pr8641
+ git -C 8641-7328d34/gopath/src/github.com/minio/minio checkout --quiet pr8641
+ GO111MODULE=on
+ GOPATH=/mnt/nvme/mint-auto/8641-7328d34/gopath
+ make -C 8641-7328d34/gopath/src/github.com/minio/minio --quiet
go: downloading github.com/minio/cli v1.22.0
go: downloading github.com/minio/minio-go v0.0.0-20190327203652-5325257a208f
go: downloading github.com/Azure/azure-storage-blob-go v0.8.0
go: downloading google.golang.org/api v0.4.0
go: downloading cloud.google.com/go v0.37.2
go: downloading github.com/fatih/color v1.7.0
go: downloading github.com/kurin/blazer v0.5.4-0.20190613185654-cf2f27cc0be3
go: downloading github.com/json-iterator/go v1.1.7
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/klauspost/compress v1.8.3
go: extracting github.com/dustin/go-humanize v1.0.0
go: downloading github.com/ncw/directio v1.0.5
go: extracting github.com/Azure/azure-storage-blob-go v0.8.0
go: extracting github.com/kurin/blazer v0.5.4-0.20190613185654-cf2f27cc0be3
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/skyrings/skyring-common v0.0.0-20160929130248-d1c0bb1cbd5e
go: extracting github.com/json-iterator/go v1.1.7
go: extracting github.com/ncw/directio v1.0.5
go: downloading github.com/djherbis/atime v1.0.0
go: extracting github.com/fatih/color v1.7.0
go: downloading github.com/klauspost/readahead v1.3.1
go: extracting github.com/minio/cli v1.22.0
go: downloading github.com/elazarl/go-bindata-assetfs v1.0.0
go: extracting github.com/minio/minio-go v0.0.0-20190327203652-5325257a208f
go: downloading github.com/minio/minio-go/v6 v6.0.39
go: extracting github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
go: extracting github.com/elazarl/go-bindata-assetfs v1.0.0
go: downloading github.com/Azure/azure-pipeline-go v0.2.1
go: extracting github.com/klauspost/readahead v1.3.1
go: extracting github.com/djherbis/atime v1.0.0
go: downloading github.com/shirou/gopsutil v2.18.12+incompatible
go: downloading go.uber.org/atomic v1.3.2
go: extracting go.uber.org/atomic v1.3.2
go: downloading github.com/coreos/etcd v3.3.12+incompatible
go: extracting github.com/shirou/gopsutil v2.18.12+incompatible
go: extracting github.com/Azure/azure-pipeline-go v0.2.1
go: downloading github.com/rjeczalik/notify v0.9.2
go: downloading github.com/beevik/ntp v0.2.0
go: extracting github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
go: extracting github.com/minio/minio-go/v6 v6.0.39
go: downloading github.com/pkg/profile v1.3.0
go: extracting github.com/klauspost/compress v1.8.3
go: downloading github.com/minio/lsync v1.0.1
go: downloading github.com/minio/mc v0.0.0-20191012041914-735aa139b19c
go: extracting github.com/pkg/profile v1.3.0
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: extracting github.com/beevik/ntp v0.2.0
go: downloading github.com/modern-go/reflect2 v1.0.1
go: extracting github.com/rjeczalik/notify v0.9.2
go: downloading github.com/mattn/go-colorable v0.1.1
go: extracting github.com/minio/lsync v1.0.1
go: downloading github.com/minio/hdfs/v3 v3.0.1
go: extracting github.com/mitchellh/go-homedir v1.1.0
go: downloading golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392
go: extracting github.com/minio/mc v0.0.0-20191012041914-735aa139b19c
go: extracting github.com/mattn/go-colorable v0.1.1
go: downloading github.com/aliyun/aliyun-oss-go-sdk v0.0.0-20190307165228-86c17b95fcd5
go: extracting github.com/modern-go/reflect2 v1.0.1
go: downloading github.com/gorilla/rpc v1.2.0
go: downloading github.com/minio/highwayhash v1.0.0
go: extracting github.com/skyrings/skyring-common v0.0.0-20160929130248-d1c0bb1cbd5e
go: extracting github.com/minio/highwayhash v1.0.0
go: downloading github.com/coredns/coredns v1.4.0
go: extracting github.com/gorilla/rpc v1.2.0
go: downloading golang.org/x/sys v0.0.0-20190922100055-0a153f010e69
go: extracting google.golang.org/api v0.4.0
go: extracting golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392
go: downloading github.com/minio/gokrb5/v7 v7.2.5
go: extracting github.com/aliyun/aliyun-oss-go-sdk v0.0.0-20190307165228-86c17b95fcd5
go: downloading github.com/minio/sha256-simd v0.1.1
go: extracting github.com/coredns/coredns v1.4.0
go: extracting github.com/minio/hdfs/v3 v3.0.1
go: downloading golang.org/x/net v0.0.0-20190923162816-aa69164e4478
go: extracting cloud.google.com/go v0.37.2
go: extracting github.com/minio/sha256-simd v0.1.1
go: downloading github.com/fatih/structs v1.1.0
go: downloading golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: downloading github.com/mattn/go-isatty v0.0.7
go: extracting golang.org/x/sys v0.0.0-20190922100055-0a153f010e69
go: downloading github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
go: downloading github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: extracting github.com/coreos/etcd v3.3.12+incompatible
go: downloading github.com/prometheus/procfs v0.0.0-20190117184657-bf6a532e95b1
go: downloading github.com/hashicorp/vault v1.1.0
go: extracting github.com/mattn/go-isatty v0.0.7
go: downloading google.golang.org/grpc v1.20.1
go: extracting golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: downloading github.com/minio/sio v0.2.0
go: extracting github.com/fatih/structs v1.1.0
go: downloading github.com/secure-io/sio-go v0.3.0
go: extracting golang.org/x/net v0.0.0-20190923162816-aa69164e4478
go: extracting github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
go: downloading github.com/klauspost/pgzip v1.2.1
go: downloading github.com/rs/cors v1.6.0
go: downloading github.com/googleapis/gax-go v2.0.0+incompatible
go: extracting github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: downloading github.com/cheggaaa/pb v1.0.28
go: extracting github.com/prometheus/procfs v0.0.0-20190117184657-bf6a532e95b1
go: downloading github.com/gorilla/mux v1.7.0
go: extracting github.com/secure-io/sio-go v0.3.0
go: downloading github.com/minio/parquet-go v0.0.0-20190318185229-9d767baf1679
go: extracting github.com/minio/sio v0.2.0
go: downloading github.com/klauspost/reedsolomon v1.9.3
go: extracting google.golang.org/grpc v1.20.1
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: extracting github.com/klauspost/pgzip v1.2.1
go: downloading github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: extracting github.com/rs/cors v1.6.0
go: downloading github.com/alecthomas/participle v0.2.1
go: extracting github.com/googleapis/gax-go v2.0.0+incompatible
go: downloading github.com/googleapis/gax-go/v2 v2.0.4
go: extracting github.com/cheggaaa/pb v1.0.28
go: downloading gopkg.in/ini.v1 v1.48.0
go: extracting github.com/gorilla/mux v1.7.0
go: extracting github.com/minio/parquet-go v0.0.0-20190318185229-9d767baf1679
go: downloading github.com/mattn/go-runewidth v0.0.4
go: extracting github.com/klauspost/reedsolomon v1.9.3
go: downloading gopkg.in/ldap.v3 v3.0.3
go: extracting github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/miekg/dns v1.1.8
go: extracting github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: downloading golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: extracting github.com/alecthomas/participle v0.2.1
go: downloading github.com/golang/protobuf v1.3.2
go: extracting github.com/googleapis/gax-go/v2 v2.0.4
go: downloading google.golang.org/genproto v0.0.0-20190307195333-5fe7a883aa19
go: extracting gopkg.in/ini.v1 v1.48.0
go: extracting github.com/mattn/go-runewidth v0.0.4
go: extracting gopkg.in/ldap.v3 v3.0.3
go: downloading github.com/mattn/go-ieproxy v0.0.0-20190805055040-f9202b1cfdeb
go: extracting github.com/miekg/dns v1.1.8
go: downloading github.com/Shopify/sarama v1.24.1
go: extracting golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: downloading golang.org/x/text v0.3.2
go: extracting github.com/golang/protobuf v1.3.2
go: downloading github.com/prometheus/common v0.2.0
go: extracting github.com/mattn/go-ieproxy v0.0.0-20190805055040-f9202b1cfdeb
go: downloading github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973
go: extracting github.com/prometheus/common v0.2.0
go: extracting github.com/Shopify/sarama v1.24.1
go: downloading gopkg.in/olivere/elastic.v5 v5.0.80
go: extracting github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973
go: downloading github.com/streadway/amqp v0.0.0-20190402114354-16ed540749f6
go: extracting github.com/streadway/amqp v0.0.0-20190402114354-16ed540749f6
go: downloading gopkg.in/jcmturner/gokrb5.v7 v7.2.3
go: extracting gopkg.in/olivere/elastic.v5 v5.0.80
go: downloading github.com/eapache/go-resiliency v1.2.0
go: extracting github.com/eapache/go-resiliency v1.2.0
go: downloading github.com/gomodule/redigo v2.0.0+incompatible
go: extracting github.com/gomodule/redigo v2.0.0+incompatible
go: downloading github.com/lib/pq v1.0.0
go: extracting github.com/hashicorp/vault v1.1.0
go: extracting github.com/lib/pq v1.0.0
go: extracting golang.org/x/text v0.3.2
go: extracting google.golang.org/genproto v0.0.0-20190307195333-5fe7a883aa19
go: downloading github.com/hashicorp/hcl v1.0.0
go: extracting github.com/hashicorp/hcl v1.0.0
go: downloading github.com/gogo/protobuf v1.2.1
go: downloading github.com/hashicorp/go-retryablehttp v0.5.3
go: downloading github.com/pierrec/lz4 v2.2.6+incompatible
go: downloading gopkg.in/yaml.v2 v2.2.2
go: extracting github.com/hashicorp/go-retryablehttp v0.5.3
go: extracting gopkg.in/yaml.v2 v2.2.2
go: downloading github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: downloading go.opencensus.io v0.21.0
go: extracting github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: extracting go.opencensus.io v0.21.0
go: extracting github.com/gogo/protobuf v1.2.1
go: extracting github.com/pierrec/lz4 v2.2.6+incompatible
go: downloading github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: extracting github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: downloading github.com/klauspost/cpuid v1.2.1
go: extracting github.com/klauspost/cpuid v1.2.1
go: downloading github.com/hashicorp/go-multierror v1.0.0
go: extracting github.com/hashicorp/go-multierror v1.0.0
go: downloading git.apache.org/thrift.git v0.12.0
go: extracting git.apache.org/thrift.git v0.12.0
go: downloading github.com/hashicorp/go-rootcerts v1.0.1
go: extracting github.com/hashicorp/go-rootcerts v1.0.1
go: extracting gopkg.in/jcmturner/gokrb5.v7 v7.2.3
go: extracting github.com/minio/gokrb5/v7 v7.2.5
go: downloading github.com/nsqio/go-nsq v1.0.7
go: downloading github.com/nats-io/stan.go v0.4.5
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/ryanuber/go-glob v1.0.0
go: extracting github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/nats-io/nats.go v1.9.1
go: extracting github.com/nats-io/stan.go v0.4.5
go: extracting github.com/ryanuber/go-glob v1.0.0
go: downloading github.com/hashicorp/go-uuid v1.0.1
go: downloading gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: extracting github.com/nsqio/go-nsq v1.0.7
go: downloading github.com/eapache/queue v1.1.0
go: extracting github.com/hashicorp/go-uuid v1.0.1
go: downloading github.com/hashicorp/go-sockaddr v1.0.2
go: extracting github.com/eapache/queue v1.1.0
go: extracting gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
go: extracting github.com/nats-io/nats.go v1.9.1
go: downloading github.com/davecgh/go-spew v1.1.1
go: extracting github.com/pkg/errors v0.8.1
go: extracting github.com/hashicorp/go-cleanhttp v0.5.1
go: downloading github.com/go-sql-driver/mysql v1.4.1
go: downloading github.com/nats-io/jwt v0.3.2
go: extracting github.com/hashicorp/go-sockaddr v1.0.2
go: extracting github.com/davecgh/go-spew v1.1.1
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: extracting github.com/nats-io/jwt v0.3.2
go: downloading github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: extracting github.com/hashicorp/errwrap v1.0.0
go: extracting github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/gorilla/handlers v1.4.0
go: extracting github.com/go-sql-driver/mysql v1.4.1
go: downloading github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: extracting github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: downloading gopkg.in/jcmturner/rpc.v1 v1.1.0
go: downloading github.com/golang/snappy v0.0.1
go: extracting github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: extracting github.com/gorilla/handlers v1.4.0
go: downloading gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: downloading github.com/eclipse/paho.mqtt.golang v1.2.0
go: extracting gopkg.in/jcmturner/rpc.v1 v1.1.0
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
go: extracting github.com/golang/snappy v0.0.1
go: extracting gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: downloading github.com/nats-io/nkeys v0.1.3
go: downloading github.com/nats-io/nuid v1.0.1
go: extracting github.com/nats-io/nuid v1.0.1
go: extracting github.com/matttproud/golang_protobuf_extensions v1.0.1
go: extracting github.com/eclipse/paho.mqtt.golang v1.2.0
go: extracting github.com/nats-io/nkeys v0.1.3
go: downloading gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading github.com/rcrowley/go-metrics v0.0.0-20190704165056-9c2d0518ed81
go: extracting gopkg.in/jcmturner/aescts.v1 v1.0.1
go: extracting github.com/rcrowley/go-metrics v0.0.0-20190704165056-9c2d0518ed81
go: downloading gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: extracting gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: finding google.golang.org/api v0.4.0
go: finding github.com/minio/minio-go/v6 v6.0.39
go: finding github.com/Azure/azure-pipeline-go v0.2.1
go: finding github.com/minio/cli v1.22.0
go: finding github.com/kurin/blazer v0.5.4-0.20190613185654-cf2f27cc0be3
go: finding github.com/dustin/go-humanize v1.0.0
go: finding github.com/aliyun/aliyun-oss-go-sdk v0.0.0-20190307165228-86c17b95fcd5
go: finding cloud.google.com/go v0.37.2
go: finding github.com/minio/sha256-simd v0.1.1
go: finding github.com/minio/gokrb5/v7 v7.2.5
go: finding github.com/minio/hdfs/v3 v3.0.1
go: finding github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: finding github.com/minio/mc v0.0.0-20191012041914-735aa139b19c
go: finding github.com/gorilla/mux v1.7.0
go: finding github.com/minio/highwayhash v1.0.0
go: finding golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: finding github.com/Azure/azure-storage-blob-go v0.8.0
go: finding github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: finding golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392
go: finding github.com/json-iterator/go v1.1.7
go: finding github.com/fatih/color v1.7.0
go: finding github.com/mitchellh/go-homedir v1.1.0
go: finding github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: finding github.com/hashicorp/go-uuid v1.0.1
go: finding github.com/mattn/go-ieproxy v0.0.0-20190805055040-f9202b1cfdeb
go: finding gopkg.in/ini.v1 v1.48.0
go: finding gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: finding golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: finding google.golang.org/genproto v0.0.0-20190307195333-5fe7a883aa19
go: finding golang.org/x/sys v0.0.0-20190922100055-0a153f010e69
go: finding go.opencensus.io v0.21.0
go: finding github.com/minio/minio-go v0.0.0-20190327203652-5325257a208f
go: finding google.golang.org/grpc v1.20.1
go: finding github.com/googleapis/gax-go/v2 v2.0.4
go: finding github.com/mattn/go-colorable v0.1.1
go: finding github.com/mattn/go-isatty v0.0.7
go: finding github.com/rjeczalik/notify v0.9.2
go: finding github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: finding github.com/golang/protobuf v1.3.2
go: finding github.com/modern-go/reflect2 v1.0.1
go: finding golang.org/x/net v0.0.0-20190923162816-aa69164e4478
go: finding github.com/gogo/protobuf v1.2.1
go: finding github.com/hashicorp/golang-lru v0.5.1
go: finding gopkg.in/jcmturner/aescts.v1 v1.0.1
go: finding github.com/secure-io/sio-go v0.3.0
go: finding gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: finding github.com/shirou/gopsutil v2.18.12+incompatible
go: finding github.com/ncw/directio v1.0.5
go: finding golang.org/x/text v0.3.2
go: finding gopkg.in/jcmturner/rpc.v1 v1.1.0
go: finding github.com/beevik/ntp v0.2.0
go: finding github.com/cheggaaa/pb v1.0.28
go: finding github.com/mattn/go-runewidth v0.0.4
go: finding github.com/coreos/etcd v3.3.12+incompatible
go: finding github.com/djherbis/atime v1.0.0
go: finding github.com/elazarl/go-bindata-assetfs v1.0.0
go: finding github.com/gorilla/handlers v1.4.0
go: finding github.com/gorilla/rpc v1.2.0
go: finding github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: finding github.com/klauspost/compress v1.8.3
go: finding github.com/klauspost/cpuid v1.2.1
go: finding github.com/klauspost/readahead v1.3.1
go: finding github.com/klauspost/reedsolomon v1.9.3
go: finding github.com/miekg/dns v1.1.8
go: finding github.com/minio/lsync v1.0.1
go: finding github.com/coredns/coredns v1.4.0
go: finding gopkg.in/ldap.v3 v3.0.3
go: finding gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: finding github.com/Shopify/sarama v1.24.1
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/eapache/go-resiliency v1.2.0
go: finding github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: finding github.com/golang/snappy v0.0.1
go: finding github.com/eapache/queue v1.1.0
go: finding github.com/pierrec/lz4 v2.2.6+incompatible
go: finding github.com/rcrowley/go-metrics v0.0.0-20190704165056-9c2d0518ed81
go: finding gopkg.in/jcmturner/gokrb5.v7 v7.2.3
go: finding github.com/eclipse/paho.mqtt.golang v1.2.0
go: finding github.com/go-sql-driver/mysql v1.4.1
go: finding github.com/gomodule/redigo v2.0.0+incompatible
go: finding github.com/lib/pq v1.0.0
go: finding github.com/nats-io/nats.go v1.9.1
go: finding github.com/nats-io/jwt v0.3.2
go: finding github.com/nats-io/nkeys v0.1.3
go: finding github.com/nats-io/nuid v1.0.1
go: finding github.com/nats-io/stan.go v0.4.5
go: finding github.com/nsqio/go-nsq v1.0.7
go: finding github.com/pkg/errors v0.8.1
go: finding github.com/skyrings/skyring-common v0.0.0-20160929130248-d1c0bb1cbd5e
go: finding github.com/streadway/amqp v0.0.0-20190402114354-16ed540749f6
go: finding gopkg.in/olivere/elastic.v5 v5.0.80
go: finding github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: finding github.com/hashicorp/vault v1.1.0
go: finding github.com/hashicorp/errwrap v1.0.0
go: finding github.com/hashicorp/go-cleanhttp v0.5.1
go: finding github.com/hashicorp/go-multierror v1.0.0
go: finding github.com/hashicorp/go-retryablehttp v0.5.3
go: finding github.com/hashicorp/go-rootcerts v1.0.1
go: finding github.com/hashicorp/hcl v1.0.0
go: finding github.com/hashicorp/go-sockaddr v1.0.2
go: finding github.com/ryanuber/go-glob v1.0.0
go: finding github.com/mitchellh/mapstructure v1.1.2
go: finding github.com/minio/sio v0.2.0
go: finding github.com/fatih/structs v1.1.0
go: finding gopkg.in/yaml.v2 v2.2.2
go: finding github.com/klauspost/pgzip v1.2.1
go: finding github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: finding github.com/alecthomas/participle v0.2.1
go: finding github.com/minio/parquet-go v0.0.0-20190318185229-9d767baf1679
go: finding git.apache.org/thrift.git v0.12.0
go: finding github.com/pkg/profile v1.3.0
go: finding github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
go: finding github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973
go: finding github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
go: finding github.com/prometheus/common v0.2.0
go: finding github.com/matttproud/golang_protobuf_extensions v1.0.1
go: finding github.com/prometheus/procfs v0.0.0-20190117184657-bf6a532e95b1
go: finding github.com/rs/cors v1.6.0
go: finding go.uber.org/atomic v1.3.2
# github.com/minio/minio/cmd/config
cmd/config/config.go:649:6: ok declared and not used
make: *** [build] Error 2

```

--- a/cmd/config/config.go
+++ b/cmd/config/config.go
@@ -644,6 +644,15 @@ func (c Config) SetKVS(s string, defaultKVS map[string]KVS) error {
 		currKVS.Set(Comment, v)
 	}
 
+	hkvs := HelpSubSysMap[subSys]
+	for _, hkv := range hkvs {
+		v, _ := currKVS.Lookup(hkv.Key)
+		if v == "" && !hkv.Optional {
+			return Errorf(SafeModeKind,
+				"'%s' is not optional for '%s' sub-system, please check '%s' documentation",
+				hkv.Key, subSys, subSys)
+		}
+	}
 	c[subSys][tgt] = currKVS
 	return nil
 }

